test/idp/src/main/kotlin/org/codice/compliance/web/sso/RedirectLoginTest.kt

```diff
@@ -35,6 +34,9 @@ import org.codice.security.sign.Encoder
 import org.codice.security.sign.SimpleSign
 import java.time.Instant
 
+/* ktlint-disable no-wildcard-imports */
+import org.apache.cxf.rs.security.saml.sso.SSOConstants.*
```

Comment:
✏️ We should import the individual `SSOConstants` instead of `suppressing?` this.

Reply:
Could the number of SSOConstants used here grow? Importing each one could get out of hand

Reply:
I don't think it will. As of right now you just need four. `SAML_REQUEST`, `SIG_ALG`, `SIGNATURE` and `RELAY_STATE`

Reply:
Okay, I would say in this case we can import each one. Some of the files it may be overkill

Reply:
I agree with both of you. In this instance, we can import the individual ones but in some edge cases it may be better to suppress.